lib/admin/services/process_file.rb

```diff
@@ -5,60 +5,46 @@ module Services
     # necessary, of an XLSX file.
     class ProcessFile
```

Comment:
I feel the naming could be improved later. The class is doing many things - validating, marshalling, parsing, coercing data.

FileOverview or something similarly named is something that's missing for me.